src/main/java/Jane.java

```diff
@@ -0,0 +1,82 @@
+import java.util.Scanner;
+public class Jane {
+    public static void processInput(String input, TaskList taskList) {
+        String[] inputPart = input.split(" ", 2);
+        Task currentTask;
+        switch (inputPart[0]) {
+        case "todo":
+            Todo todo = new Todo(inputPart[1]);
+            taskList.addTask(todo);
+            System.out.println("Got it. I've added this task:\n" + todo + "\n" +
+                    "Now you have " + taskList.getCount() + " tasks in the list.");
+            break;
+        case "deadline":
+            String[] deadlineInput = inputPart[1].split("/", 2);
```

Comment:
might be better if the variables declaration are all grouped together before the switch case starts.